bitcoin/embedded/src/main.rs

```diff
@@ -7,7 +7,6 @@ extern crate alloc;
 extern crate bitcoin;
```

Comment:
FTR these are not clippy warnings but rustc warnings. Please don't bother with fixing the commit message, this is blocking other work.